bert_e/tests/unit/test_github_build_status.py

```diff
@@ -111,11 +133,14 @@ def test_aggregated_workflow_run(client, workflow_run_json):
 
     workflow_run_json['workflow_runs'][0]['status'] = 'queued'
     workflow_run_json['workflow_runs'][0]['conclusion'] = None
+    workflow_run_json['workflow_runs'][2]['status'] = 'queued'
+    workflow_run_json['workflow_runs'][2]['conclusion'] = None
```

Comment:
Forgive me, I don't fully remember the whole test context here but don't we expect this particular workflow to stay the same, with `status: completed` and `conclusion: skipped` and aggregatedWorkflowRuns should consider it as a success. If we are modifying it's state I'm not sure we are testing what we are fixing.

And if what you are testing is the same `workflow_id` one with skipped it's not exactly the same scenario we had. We have two different workflow, one completely skipped and one successful.